app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<section class="depot_form">
+  <% if flash[:alert] %>
+    <aside class="notice"><%= flash[:notice] %></aside>
```

Comment:
Mismatch of `flash[:alert]` and `flash[:notice]`. Not sure if that's intended?

Reply:
FIXED